packages/teams-js/src/public/runtime.ts

```diff
@@ -287,7 +290,7 @@ export const upgradeChain: IRuntimeUpgrade[] = [
     upgradeToNextVersion: (previousVersionRuntime: IRuntimeV1): IRuntimeV2 => {
       return {
         apiVersion: 2,
-        hostVersionsInfo: undefined,
+        hostVersionsInfo: previousVersionRuntime.hostVersionsInfo,
```

Comment:
hostVersionsInfo isn't on IRuntimeV1, how does this work?

Reply:
I was testing it out and forgot to revert it back. Updated to undefined in latest commit.